app/stirr/views/view.html

```diff
@@ -39,8 +39,16 @@
         </label>
       </div>
     </div>
-    <button class="button button-full button-positive" ng-click="forkRecipe()">
+    <div class="button-bar">
```

Comment:
The corners of the button bar are ever so slightly rounded (at least on Android) and it looks bad. Can we either add margins around the button bar or make it borderless? Also, are we keeping the button down here for now?